xds/internal/balancer/ringhash/util.go

```diff
@@ -18,23 +18,24 @@
 
 package ringhash
 
-import "context"
+import (
+	"context"
+)
 
-type clusterKey struct{}
+type xdsHashKey struct{}
 
-func getRequestHash(ctx context.Context) uint64 {
-	requestHash, _ := ctx.Value(clusterKey{}).(uint64)
-	return requestHash
+// GetXDSRequestHash returns the request hash in the context, set from the
+// xDS config selector.
```

Comment:
nit: this comment should mention what is returned if the request hash does not exist in the context.